tools/redfish-gen/redfish_gen/redfish_schema.py

```diff
@@ -0,0 +1,543 @@
+## SPDX-License-Identifier: Apache-2.0
```

Comment:
Looks like you have copied the redfish_schema.py from redfish_node.py, and now we have there are too many code-repeats.
Would it be better to import already existing redfish_node.py and inherit RedfishNode class with overriding required methods?

I have high doubt the way you picked is correct.

Reply:
As discussed with @ikmsk10, I created the RedfishBase class with command functionality as super class for the RedfishNode and RedfishSchema.